src/langs/en.json

```diff
@@ -100,9 +100,9 @@
   "ITEM_CATEGORIES_CONTAINER_TITLE": "Category",
   "ITEM_CATEGORIES_CONTAINER_MISSING_WARNING": "Add at least one category to better define the global theme of your content",
   "ITEM_CATEGORIES_CONTAINER_EMPTY_BUTTON": "Add a category",
-  "ITEM_LANGUAGES_CONTAINER_TITLE": "Languages",
-  "ITEM_LANGUAGES_CONTAINER_MISSING_WARNING": "Add at least one language in which your publication is written",
-  "ITEM_LANGUAGES_CONTAINER_EMPTY_BUTTON": "Add a language",
+  "ITEM_LANGUAGE_CONTAINER_TITLE": "Language",
+  "ITEM_LANGUAGE_DESCRIPTION": "Choose the language which your item is written in.",
+  "ITEM_LANGUAGE_DESCRIPTION_LIMITATION": "Graasp proposes a fixed set of languages. If your language is not in the set, you should use English.",
```

Comment:
Maybe we should let them know that they can contact us and we can discuss the possibility of adding support for a new language ? The current formulation feels a bit rough. Not very inclusive for other languages I would say.